### DISCUSSION TITLE
若依vue集成jsmind时报解析错误

### DISCUSSION THREAD

**Discussion by longhaolh** [hizzgdev/jsmind#583] [General]
报错信息: error  in ./node_modules/jsmind/es6/jsmind.js

Module parse failed: Unexpected token (8:36865)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file,
我尝试添加解析器之后还是无法正常解析,请问又遇到一样问题的吗?
![image](https://github.com/hizzgdev/jsmind/assets/51143957/1ac47f28-c195-407e-9352-fe85153efb06)
![image](https://github.com/hizzgdev/jsmind/assets/51143957/a481e765-df79-496a-b8a7-e3cd8e773bc2)
![image](https://github.com/hizzgdev/jsmind/assets/51143957/94ecf230-eed6-4d03-9d0b-3a7bc4f5a9b3)

**Comment by hizzgdev** [comment]
https://github.com/hizzgdev/jsmind-samples/tree/main/jsmind-vue

可以参考一下这个 demo，如果不行的话，你看能否提供一个简易的测试项目，我可以一起帮看一下。

**Reply by longhaolh** [reply]
谢回，若依vue用的是vue2 不知道是不是因为这个原因，现在报缺少依赖了
![image](https://github.com/hizzgdev/jsmind/assets/51143957/4a86243d-2aef-40d3-a92d-54e381c62bf7)

**Reply by longhaolh** [reply]
修复完路径还是报解析错误
![image](https://github.com/hizzgdev/jsmind/assets/51143957/99e6f70d-6f30-42b0-a8ec-0c46c2a591e0)
![image](https://github.com/hizzgdev/jsmind/assets/51143957/aa401d9f-7fdc-452d-8bd7-5462195d3f80)
暂时还不知道是哪里的问题.我也只是拿来测试用的 若依vue部署完成集成jsmind就出现这个问题 不引入jsmind没有报错

**Reply by hizzgdev** [reply]
按说应该这么用, 但你这里第一个 import 就好像有问题。
```es6
import jsMind from 'jsmind'
import 'jsmind/draggable-node'
import 'jsmind/style/jsmind.css'
```
我还不了解 若依vue ，后面有时间我也试一下，你也可以到 若依vue 的论坛上问一下，如果有什么反馈也可以随时同步到这里来，谢谢。

**Reply by hizzgdev** [reply]
我看也有 Vue3 的版本，你试过吗？ https://github.com/yangzongzhuan/RuoYi-Vue3

**Reply by longhaolh** [reply]
抱歉这几天没看信息,引入的没错,点击变量都能找到源代码,主要是运行的时候报解析错误,不知道是为什么,在其它vue项目中可以正常解析和使用,怀疑是ruoyi的配置问题

**Reply by tangbing777** [reply]
请问有解决方案了吗 我也遇到了 在vue中打包报错You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See
https://webpack.is.org/concepts#loaders
水
https://github.com/hizzadev/ismind/
*/

**Reply by yimi-fighting** [reply]
请问有解决方案了吗，我也遇到了这个问题

**Reply by hizzgdev** [reply]
你们有谁能在github或gitee上提交一个简易的测试项目吗，我可以一起帮看一下。
